app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_to_buy = 0
+
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return (
+                "All friends should be vaccinated"
+            )
+        except NotWearingMaskError:
+            mask_to_buy += 1
+
+    if mask_to_buy != 0:
+        return (
+            f"Friends should buy {mask_to_buy} masks"
+        )
+    else:
+        return (
+            f"Friends can go to {cafe.name}"
+        )
```

Comment:
You can simplify this part of code using `return <some text> if condition else <some text>`